docs/topics_5/web-create-project.adoc

```diff
@@ -0,0 +1,153 @@
+// Module included in the following assemblies:
+// * docs/web-console-guide_5/master.adoc
+[id='add_project_{context}']
+= Creating a project
+
+You create a project using a wizard-like interface.
+
+.Procedure
+
+. In the navigation pane, click *Projects*.
+. On the *Projects* page, click *Create project*.
++
+.Create project
+image::web-add-project_51.png[Create project]
+
+. Enter a unique name for the project.
+. Optional: Enter a description.
+. Click *Next*.
++
+.Add applications
+image::web-add-apps_51.png[Add applications]
++
+You can upload applications or register the server path that contains an application.
+
+. To upload applications:
+
+ .. In the *Upload* tab, click *Browse*.
+ .. Select the application files you want to upload.
+ .. Click *Close*.
+
+. To register a server path:
+
+ .. Click the *Server path* tab.
+ .. Enter the *Labels path* of the server-side file or directory containing the application in the field.
+ .. Select *Directory is an exploded Java application archive* if relevant.
+
+ . Click *Next*.
++
+[NOTE]
+====
+Uploading applications stores them directly in the {ProductShortName} server.
+
+Registering the directory path of an application tells the {ProductShortName} server the location of the application. This method is useful if you expect to update the application because it ensures that the {ProductShortName} server always uses the latest version of the application.
+====
++
+.Select transformation targets
+image::web-configure-analysis_51.png[Select transformation targets]
+
+. Click the desired transformation targets.
+. Click *Next*.
++
+You can specify which Java application packages to include or exclude on the *Select applications* page.
++
+The panel on the left contains all of the identified packages in the project. The panel on the right contains only the packages that are to be analyzed.
+
+. Select the packages you want to include or exclude and click *>* or *<*.
+. Click *Next*.
++
+.Custom rules
```

Comment:
Are custom rules required or optional? If they're optional, I would write:

If you want to add custom rules, click Add rule.